src/app/clusters/time-synchronization-server/time-synchronization-server.cpp

```diff
@@ -570,20 +564,17 @@ CHIP_ERROR TimeSynchronizationServer::SetTimeZone(const DataModel::DecodableList
         const auto & newTz = newTzL.GetValue();
         if (newTz.offset < -43200 || newTz.offset > 50400)
         {
-            ReturnErrorOnFailure(LoadTimeZone());
-            return CHIP_ERROR_IM_MALFORMED_COMMAND_DATA_IB;
+            ExitNow(err = CHIP_IM_GLOBAL_STATUS(ConstraintError));
```

Comment:
so some of these error messages have changed - do tests not complain about this? Are we not testing this? Or is this caught and changed elsewhere?

Reply:
These errors are checked in the ember callbacks and that has been updated. So it doesn't bring change in the overall outcome.